client/src/redux/donation-saga.js

```diff
@@ -45,7 +45,7 @@ function* showDonateModalSaga() {
     if (recentlyClaimedBlock) {
       yield put(preventBlockDonationRequests());
     } else {
-      yield put(preventProgressDonationRequests());
+      yield put(setSessionChallNumProgressModalShown());
```

Comment:
This variable is a little confusing to me. 

My first thought is that it should just be "setProgressDonationModalShown". I think that's clear enough.

Also, I think it would be clearer if the reducer set two properties in the store: `progressDonationModalShown` and `completionCountWhenShownProgressModal`. They're a bit wordier than I'd like, but there we are.

If we use these two, it should make it a little clearer what's going on. I'll update the selector logic with an example of what I mean.

Reply:
The last commit should have resolved this.

Reply:
The last commit should have resolved this.